specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Domain Model analysis request": {
+                        "$ref": "./examples//SuccessfulDomainModelWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/recognizeText?overload=url": {
+            "post": {
+                "description": "Recognize Text operation. When you use the Recognize Text interface, the response contains a field called “Operation-Location”. The “Operation-Location” field contains the URL that you must use for your Get Handwritten Text Operation Result operation.",
+                "operationId": "RecognizeText",
+                "parameters": [
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/ImageUrl"
+                    },
+                    {
+                        "$ref": "#/parameters/HandwritingBoolean"
+                    }
+                ],
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "202": {
+                        "description": "The service has accepted the request and will start processing later. It will return Accepted immediately and include an Operation-Location header. Client side should further query the operation status using the URL specified in this header. The operation ID will expire in 48 hours.",
+                        "headers": {
+                            "Operation-Location": {
+                                "description": "URL to query for status of the operation. The operation ID will expire in 48 hours. ",
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Domain Model analysis request": {
+                        "$ref": "./examples//SuccessfulRecognizeTextWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/textOperations/{operationId}": {
+            "get": {
+                "description": "This interface is used for getting text operation result. The URL to this interface should be retrieved from 'Operation-Location' field returned from Recognize Text interface.",
+                "operationId": "GetTextOperationResult",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages